src/controller/storagemgr/storage.go

```diff
@@ -0,0 +1,46 @@
+package storagemgr
```

Comment:
Please put the copyright statement at the head.

Reply:
Ok. I will do that.